app/tests/groups_tests/test_views.py

```diff
@@ -52,9 +50,6 @@ def get_user_autocomplete():
                 data={"q": u.username.lower()},
             )
 
-        response = get_user_autocomplete()
-        assert response.status_code == 403
-
```

Comment:
Does this now return HTTP 200 and an empty list? If so, it would be better to check for that explicitly I think in case something is changed later and it accidentally returns non empty.